db/migrate/20230712010439_create_users.rb

```diff
@@ -9,4 +9,4 @@ def change
       t.timestamps
     end
   end
-end
+end
```

Comment:
Please permit me to use this line to request changes to your README file.

- [ ] Kindly fix the LICENSE link on your README file, the current link is broken.

Reply:
Thank you, I am creating the License file now.